joinarrays.rb

```diff
@@ -0,0 +1,30 @@
+#!/urs/bin/ruby -w
+
+arr1 = Array[2, 4, 6, 8, 10]
+arr2 = Array[1, 3, 5, 7, 9, 1, 13]
+
+all = []
+
+arr1.each(){ |i|
+  all.push(i)
+}
```

Comment:
1. No need to write empty brackets `()` with method calls
2. if you are writing block in multiple lines then use `do..end` syntax and if you are using `{}` syntax then write the logic in single line
```suggestion
arr1.each { |i| all.push(i) }
```
3. Add space before `{`